plat/imx/imx8m/imx8mm/imx8mm_bl31_setup.c

```diff
@@ -204,6 +204,10 @@ void bl31_platform_setup(void)
 	plat_gic_init();
 
 	imx_gpc_init();
+
+	/* Enable and reset M4 */
+	mmio_setbits_32(IMX_SRC_BASE + SRC_M4RCR_OFFSET, SRC_ENABLE_M4);
+	mmio_clrbits_32(IMX_SRC_BASE + SRC_M4RCR_OFFSET, SRC_SW_M4C_NON_SCLR_RST);
```

Comment:
This is duplicated with:
		val = mmio_read_32(IMX_SRC_BASE + SRC_M4RCR_OFFSET);
		val &= ~SRC_SW_M4C_NON_SCLR_RST;
		val |= SRC_ENABLE_M4;
		mmio_write_32(IMX_SRC_BASE + SRC_M4RCR_OFFSET, val);

 tested on imx8mm EVK, only, get "## Auxiliary core is already up".